Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityConfigurationProfile.asset

```diff
@@ -16,4 +16,8 @@ MonoBehaviour:
     classReference: Microsoft.MixedReality.Toolkit.InputSystem.MixedRealityInputManager,
       Microsoft.MixedReality.Toolkit.InputSystem
   enableControllers: 1
+  leftControllerModel: {fileID: 1774971220009634, guid: 89aca206fbe292649bca7605a95fa801,
```

Comment:
please forgive if this is a dumb question... are we moving away from getting the model from the controller firmware?

Reply:
That's correct we will be loading the model from the driver, but we still want to give devs the option to use their own custom model if they so choose.